llama_stack/providers/inline/inference/sentence_transformers/config.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from pydantic import BaseModel
+
+
+class SentenceTransformersInferenceConfig(BaseModel): ...
```

Comment:
this has no `model` field and yet the impl looks for it. BUG!

Reply:
ah in the check_model function? that function is unused. removing it.